CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Contributing Guide
+
+First off, thanks for taking the time to contribute!
+
+The following is a short set of guidelines for contributing to Iroha.
+
+## How Can I Contribute?
+
+### TL;DR
+
+* Find Jira
```

Comment:
Find a Jira issue – you can start with one of the [Good First Issues](https://jira.hyperledger.org/browse/IR-80?jql=project%20%3D%20IR%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20good-first-issue%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC)